Gruntfile.js

```diff
@@ -0,0 +1,347 @@
+            return '/' + value;
+        });
+
+        grunt.log.writeln('Purging cache for host ' + fastly.plugins.options.host);
+        grunt.log.writeln('Purging cache for urls ' + fastly.plugins.options.urls);
+
+        // Set the config back.
+        grunt.config.set('fastly', fastly);
+    });
+
+    grunt.registerTask('clean','Cleans build artifacts',function(){
+        grunt.file.delete('dist/');
+        grunt.log.writeln('Removed '+'dist/'.green);
+    });
+
+    grunt.registerTask('rename','Renames build artifacts to include version number',function(){
+        // If pushing to develop or a feature branch, codeship sends in the branch name so that builds can
+        // be pushed to S3 with the branch name included to assist multi-branch development
+        var branchFileModifier = '';
+        if(grunt.option('branch')) {
+            branchFileModifier = '-' + grunt.option('branch').replace('/', '-');
+        }
+
+        var version = String(grunt.file.readJSON('package.json').version);
+        var versionArray = version.split('.');
+        if (versionArray.length < 3) {
+            grunt.fail.warn('Version should be specified in Major.Minor.Patch (1.0.0)');
+        }
+        var majorVersion = versionArray[0];
+
+        var plugins = ['eloqua'];
+        _.each(plugins, function(type){
+            /**
+             * Output files for specific Version (when version = 1.3.0)
+             * These should never overwrite files in our s3 bucket, always should be unique.
+             * Code changes require update to version.
+             *
+             * plugins/eloqua/main-1.0.0.js
+             * plugins/eloqua/main-1.0.0.gz.js
+             */
+            grunt.file.move('dist/plugins/' + type + '/main.js', 'dist/plugins/' + type + '/' + branchFileModifier + '-' + version + '.js');
+            grunt.file.move('dist/plugins/' + type + '/main.js.gz', 'dist/plugins' + type + '/' + branchFileModifier + '-' + version + '.js.gz');
+
+            /**
+             * Output files for Major Version (when version = 1.3.0 -- majorVersion = 1)
+             * These replace the existing major release version.  The concept is that customers
+             * can use v1 and we can update it with non-breaking changes without them having to update their
+             * source.
+             *
+             * plugins/eloqua/main-v1.js
+             * plugins/eloqua/main-v1.gz.js
+             */
+            grunt.file.copy('dist/plugins/' + type + '/' + branchFileModifier + '-' + version + '.js', 'dist/plugins/' + type + '/' + branchFileModifier + '-v' + majorVersion + '.js');
+            grunt.file.copy('dist/plugins/' + type + '/' + branchFileModifier + '-' + version + '.js.gz', 'dist/plugins/' + type + '/' + branchFileModifier + '-v' + majorVersion + '.js.gz');
+
+        });
+    });
+
+
+    grunt.registerMultiTask('md5', 'Generates MD5 for build artifacts', function() {
+        this.files.forEach(function(file) {
```

Comment:
Now that we have lodash, could you do a quick search and replace of all the `forEach` blocks?